en/docs/learn/username-recovery-via-user-preferred-notification-channel.md

```diff
@@ -0,0 +1,353 @@
+                    "from"="NEXMO",
+                    "to"={{mobile}},
+                    "text"={{body}}
+                    }</inline>
+            </mapping>
+            <to eventAdapterType="http">
+                <property name="http.client.method">httpPost</property>
+                <property name="http.url">https://rest.nexmo.com/sms/json</property>
+            </to>
+        </eventPublisher>
+        ``` 
+        
+        !!! note
+            This publisher uses NEXMO as the SMS REST service provider. For more information 
+            on writing a custom http event publisher, see [HTTP Event Publisher](https://docs.wso2.com
+            /display/DAS300/HTTP+Event+Publisher).  
+            
+4.  [Start WSO2 IS](../../setup/running-the-product#starting-the-server)
+    and [log in to the management console](../../setup/running-the-product#accessing-the-management-console)
+    : ` https://<IS_HOST>:<IS_PORT>/carbon `.  
+    **NOTE:** If your IS is already running, make sure to restart to apply above configurations. 
+    
+5.  Navigate to `Main -> Identity Providers -> Resident -> Account Management Policies` section.
+
+6.  Expand the `Account Recovery` section and configure the following properties. 
+
+    ![username-recovery-configs](../assets/img/learn/account-recovery/username-recovery/username-recovery-configs.png)
+    
+    <table>
+        <thead>
+            <tr class="header">
+                <th>Field</th>
+                <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr class="odd">
+                <td>Enable Username Recovery</td>
+                <td>Enable username recovery for the current tenant domain</td>
+            </tr>
+            <tr class="odd">
+                <td>Enable Internal Notification Management</td>
+                <td>
+                    <p>
+                        Select to configure Identity server to send confirmation emails to the user.
+                        If the client application handles notification sending already, unselect it. 
+                    </p>
+                </td>
+            </tr>
+        </tbody>    
+    </table>
+
+## Managing Notification templates
+
+### Managing EMAIL Notification templates
+
+The email notification templates are stored in the `IS_HOME>/repository/conf/email/email-admin-config.xml` 
+file and they can be edited using the Management Console.
+
+!!! tip
+    The **AccountIdRecovery** template is used to send email notifications. You can edit and 
+    customize the email template. For more information, 
+    see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+    
+### Managing SMS Notification templates      
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to edit the
+sms notification template.
+
+1. Log in to the Management Console and click `Main> Registry> Browse`.
```

Comment:
Main> Registry> Browse should be in bold and not monospace. 

please apply to all applicable places